compose.yml

```diff
@@ -13,6 +13,8 @@ services:
         condition: service_healthy
     volumes:
       - db_data:/var/lib/postgresql/data
```

Comment:
nitpick(unrelated to this PR): this volume mount looks weird:

- There is no ["top level `volumes` directive"](https://docs.docker.com/compose/compose-file/compose-file-v3/#volume-configuration-reference) referencing the named volume `db_data`
- It's not clear why the application would have direct access to postgres "raw" data on the file system

Did I miss something?

Reply:
Yes, this is something I forgot to declare here, and that is why the db content is erased at each restart of the db service.